### PR TITLE
Add orgchart plugin

### PR DESCRIPTION
**Pull request by samuel-soubeyran**
Port the orgchart pluging for sigma to the sigma repo

### CODE REVIEW

File: orgchart/src/App.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  const name = config.name;
+  const id = config.id;
+  const parentId = config.parentId;
+  const positionName = config.positionName; //config.positionName;
+  const area = config.area; //config.area;
+  const imageUrl = config.imageUrl;// config.imageUrl;
+  const location = config.location; //config.location;
+
+  //console.log('got data', config, sigmaData, columns, name, id, parentId);
+  //console.log(config.source);
```
Comment (jsjqian) [inline orgchart/src/App.js:89]: Do we need these comments?

File: orgchart/src/App.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  const columns = useElementColumns(config.source);
+  const sigmaData = useElementData(config.source);
+  console.log("source", config.source);
+  console.log("data", sigmaData);
+  console.log("columns", columns);
+  /*  sigmaData = {}
+    sigmaData[config.name] = ['sam', 'anand', 'eran', 'darien']
+    sigmaData[config.parentId] = ['anand', 'eran', null, 'eran']
+    sigmaData[config.id] = ['sam', 'anand', 'eran', 'darien']
+  */
```
Comment (jsjqian) [inline orgchart/src/App.js:138]: Do we still need these?

**Review by jsjqian — APPROVED** [review]
LGTM, some clean up nits

**Review by wtgjxj — APPROVED** [review]
Probably don't need the react logos in here

**Pull request merged**